<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#73C2FB;</string>
      </property>
      <property name="text">
       <string>         Password Generator</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>19</number>
       </property>
       <property name="leftMargin">
        <number>60</number>
       </property>
       <property name="rightMargin">
        <number>31</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_standard">
         <property name="styleSheet">
          <string notr="true">QLabel {
    font-size: 12px;
    border-radius: 20px;
    background-color: #73C2FB;
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_standard">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    font-size: 12px;
    font-weight: bold;
    border-radius: 20px;
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #73C2FB, stop:1 white);
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
         </property>
         <property name="text">
          <string>Standard strenth password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>60</number>
       </property>
       <property name="rightMargin">
        <number>31</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_strong_pass">
         <property name="styleSheet">
          <string notr="true">QLabel {
    font-size: 12px;

    border-radius: 20px;
    background-color: #73C2FB;
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_strong_pass">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    font-size: 12px;
    font-weight: bold;
    border-radius: 20px;
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #73C2FB, stop:1 white);
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
         </property>
         <property name="text">
          <string>Extra Strong Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>34</number>
       </property>
       <property name="leftMargin">
        <number>60</number>
       </property>
       <property name="rightMargin">
        <number>31</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_super_pass">
         <property name="styleSheet">
          <string notr="true">QLabel {
    font-size: 12px;

    border-radius: 20px;
    background-color: #73C2FB;
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_super_pass">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    font-size: 12px;
    font-weight: bold;
    border-radius: 20px;
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                      stop:0 #73C2FB, stop:1 white);
    color: black;
    border: 2px solid rgba(0, 0, 0, 100);
    padding: 15px;
}</string>
         </property>
         <property name="text">
          <string>Super Strong Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
